### PR TITLE
slack sdk change

### LINKED ISSUE #41: Upgrade the Slack SDK
It looks like we're using an old version of an unmaintained SDK (see: https://github.com/bluele/slack), which is now archived and recommending https://github.com/slack-go/slack

### PR DESCRIPTION
**Pull request by ManishaKumari295**
Closed https://github.com/sensu/sensu-slack-handler/issues/41

**Description**
Update slack dependent sdk from "github.com/bluele/slack" to "github.com/slack-go/slack" and its related functions in code

**Change verification**
Added snippet of message sent to slack :
Screenshot 2024-05-30 at 4 20 19 PM

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -5,10 +5,11 @@ import (
 	"os"
 	"strings"
 
-	"github.com/bluele/slack"
+	//"github.com/bluele/slack"
```
Comment (fguimond) [inline main.go]: Nitpick: commented out code should be removed :)

File: main.go [hunk 2]
```diff
@@ -220,15 +221,16 @@ func messageAttachment(event *corev2.Event) *slack.Attachment {
...
-		IconUrl:     config.slackIconURL,
+		IconURL:     config.slackIconURL,
 		Username:    config.slackUsername,
-	})
+	}
+
+	err := slack.PostWebhook(config.slackwebHookURL, hookmsg)
 	if err != nil {
-		return fmt.Errorf("Failed to send Slack message: %v", err)
+		return fmt.Errorf("Failed to send Slack message: %v ", err)
```
Comment (fguimond) [inline main.go]: I think you accidentally added an extra white space at the end of the caption.

**Comment by fguimond** [comment]
Aside from a couple of small comments it looks good and should be ready to be merged.

**Review by fguimond — APPROVED** [review]
LGTM

**Comment by boutetnico** [comment]
Please publish on bonsai. 🙏🏻

**Pull request merged**